public/app/features/dashboard/dashgrid/PanelStateWrapper.test.tsx

```diff
@@ -137,6 +137,70 @@ describe('PanelStateWrapper', () => {
       });
     });
   });
+
+  describe('Panel row', () => {
+    it('should call query runner when it is not a row panel', async () => {
+      const subject: ReplaySubject<PanelData> = new ReplaySubject<PanelData>();
+
+      const panelQueryRunner = {
+        getData: () => subject,
```

Comment:
@juanicabanas why is the test on PanelStateWrapper when the fix was in PanelModel?

Reply:
In the beginning I thought PanelStateWrapper had to have the change, but I was wrong.
I can remove this test in another PR

Reply:
@juanicabanas it is a very strange but, panel rendering route is not for rows